src/utils/chunkref.c

```diff
@@ -82,14 +82,14 @@ void *nn_chunkref_getchunk (struct nn_chunkref *self)
 
     if (self->u.ref [0] == 0xff) {
         ch = (struct nn_chunkref_chunk*) self;
-        self->u.ref [0] = 0;
+        // self->u.ref [0] = 0;
```

Comment:
I could not find the reason why this is here. It looks that when somebody calls `nn_chunkref_getchunk` the internal reference is just released. This makes no sense, since there is no other way to get the internal chunk and there might be more than one call needed.